### ISSUE TITLE
Is this "project" alive?

### ISSUE THREAD

**Issue by SamMousa** [bunq/sdk_php#227]
I realize that the PHP sdk consists mostly of generated code, that is not what this is about.

There are a lot of open issues that seem to get no attention from anyone at Bunq.
In the latest version of the generated code there are issues with the library of classes extending non-existing classes, things that can be easily caught by automated static analysis of the generated code.

Is there any desire to make this a mature SDK or is it actually better to just use the well documented API and roll your own API client?

**Comment by d0x7** [comment]
They don't give a shit about this at all. Once in a while, they accept PRs but that's about it. Both this API and Java are completely broken by their bot. Their bot always does something, I guess somehow regenerating it according to current OpenAPI spec, but that does not work with the code that was written manually, hence breaking it. Both Java and PHP are missing classes such as CardPrimaryAccountNumber (used in Card.php#150 for PHP and Card.java#156) or for the Java SDK, there's a class called NotificationFilterPushUserInternal which, according to README should be used instead of the generated NotificationFilterPushUser, but the NotificationFilterPushUser doesn't even exist because the Bot deleted it ~15 versions ago and therefore the last ~15 Jitpack builds are constantly failing because of missing classes. It's a big, big mess honestly.

**Comment by SamMousa** [comment]
Yup, we're planning on sunsetting our integration with Bunq as soon as we can.

**Comment by d0x7** [comment]
Oh wow, that was a quick reply. Yeah wish I could do the same. I'm not using in a business way, but because it's my private bank. And for a small webapp, I need to access the bunq API. I've used the PHP one before and it's still in use in my project and still working, but it seems the newer versions are patched broken as I just said. I'm rewriting said app as a microservice in java right now, therefore I need the Java API now. I'm mostly just removing the broken references as I don't personally need them, but it's still stupid. I might try to generate a brand new client from the API spec, check what's not working with a generated client, fix that and upload it to GH, but I most likely won't have the time to keep this always updated when they change their spec - unless it's breaking my application.

**Comment by two-trick-pony-NL** [comment]
Hi! We're closing this issue as it's tied to an older version of our SDKs, and we've since completely revamped them as part of bunq Update 28.

This is a fresh start—we've tested the new SDKs extensively, and we’ll be putting them to the test again during our upcoming hackathon this Friday, where we’ll fix idiosyncrasies on the spot. (You’re welcome to [join us](https://www.bunq.com/en-nl/hackathon)!)

**Comment by SamMousa** [comment]
Could you leave a link here pointing to this new SDK?

**Comment by two-trick-pony-NL** [comment]
@SamMousa the SDK is this github repo, we released version 1.28 last night and you could automatically add it by running 
`$ composer require bunq/sdk_php`
